app/models/card.rb

```diff
@@ -1,6 +1,77 @@
 require "super_memo"
 
 class Card < ActiveRecord::Base
+  include Swagger::Blocks
+  swagger_schema :Card do
+    key :id, :Card
+    key :required, [
+                     :id,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.